ARCs/arc-0014.md

```diff
@@ -0,0 +1,246 @@
+---
+arc: 14
+title: Authentication with Algorand accounts
+description: Use Algorand accounts to authenticate with third-party services
+author: Stefano De Angelis (@deanstef)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/42
+status: Draft
+type: Meta
+created: 2022-04-05
+---
+
+# Authentication with Algorand accounts
+
+A standard approach to authenticate users with Algorand accounts.
+
+## Abstract
+
+This ARC introduces a standard for SSO authentication based on Algorand accounts. It leverages the public-secret key <*PK, SK*> encryption schema to verify the identity of a user that owns and Algorand account. This approach fosters the adoption of novel identity management systems for Web3 applications.
```

Comment:
Typo: 
`a user that owns and Algorand account` -> `a user that owns an Algorand account`